src/dashboard/src/Layout/NotificationBox.tsx

```diff
@@ -30,29 +31,58 @@ const usePaperStyle = makeStyles(theme => createStyles({
   }
 }))
 
-const NotificationBox: FunctionComponent<BoxProps> = (props) => {
-  const { notifications } = useContext(ConfigContext)
-  const { currentTeamId } = useContext(TeamContext)
-
-  const notification = useMemo(() => {
-    const notification = get(notifications, [currentTeamId])
-    if (notification === undefined) {
-      return get(notifications, ['.default'])
-    }
-    return notification
-  }, [notifications, currentTeamId])
+const ClusterNotificationBox: FunctionComponent<{ cluster: any, setAmlLink: any}> = ({ cluster, setAmlLink }) => {
+  const { data } = useFetch(`/api/clusters/${cluster.id}`, [cluster.id])
+  const notifications = useMemo(() => {
+    if (data === undefined) return []
+    if (!Array.isArray(data.notifications)) return []
+    if (data.amlPortal !== undefined) setAmlLink(data.amlPortal)
+    return data.notifications as string[]
+  }, [data, setAmlLink])
 
   const paperStyle = usePaperStyle()
 
-  if (notification == null || notification === '') return null
+  if (data === undefined) return null
 
+  return (
+    <>
+      {
+        notifications.map((notification, index) => (
+          <Fragment key={index}>
+            <Paper elevation={0} classes={paperStyle}>
+              <Info fontSize="small" color="primary"/>
+              <Typography
+                variant="body2"
+                component={Box}
+                flex={1}
+                paddingRight={1}
+                dangerouslySetInnerHTML={{ __html: notification }}
+              />
+            </Paper>
+            <Divider/>
+          </Fragment>
+        ))
+      }
+    </>
+  )
+}
+
+const NotificationBox: FunctionComponent<BoxProps> = (props) => {
+  const { clusters } = useContext(ClustersContext)
+  const [amlLink, setAmlLink] = useState(undefined)
   return (
     <Box {...props}>
-      <Paper elevation={0} classes={paperStyle}>
-        <Info fontSize="small" color="primary"/>
-        <Typography variant="body2" component={Box} flex={1} paddingLeft={1}>{notification}</Typography>
-      </Paper>
-      <Divider/>
+      {clusters.map(cluster => (
+        <ClusterNotificationBox key={cluster.id} cluster={cluster} setAmlLink={setAmlLink}/>
+      ))}
+      {amlLink
+        ? <Grid item xs={12} container justify="flex-end">
+          <Info fontSize="small" color="primary"/>
+          <Tooltip title="New experimental features. Global job scheduler enables running job on underutilized GPU capacity from other teams. Elastic training enables running a training job in a fault-tolernat and elastic manner.">
+            <Link href={amlLink} target="_blank" underline='none'>Try global job scheduler and elastic training</Link>
```

Comment:
add `rel="noopener noreferrer"` to link to fix the vulnerabilities.

It should be issued by linter, will check it later